README.md

```diff
@@ -30,34 +36,50 @@ provider "cyral" {
 provider "okta" {
   org_name  = "your-organization-name"
   base_url  = "okta.com" # your organization url
-  api_token = "your-api-token"
+  api_token = ""
 }
 
-module "cyral-idp-okta" {
+module "cyral_idp_okta" {
   source = "cyralinc/idp-okta/cyral"
   version = ">= 2.0.0"
 
   control_plane = "mytenant.cyral.com:8000"
-  integration_name = "Okta SSO"
-
+  
   okta_app_name = "Cyral"
   okta_groups = ["Everyone"]
+
+  idp_integration_name = "Okta SSO"
+  back_channel_logout = false
+  disable_force_authentication = true
+  allowed_clock_skew = 0
```

Comment:
I would like to simplify the module as much as possible. Are these variables that we can assign default values and let them out of the basic deployment routine? I'm fine to make them mandatory as long as they are also mandatory for Okta or if they are a big security concern.

Reply:
Yes, those variables are optional and by default they have the value retrieved from the metadata. Customers can still use the module without those variables, with a simple configuration like:
```
module "cyral_idp_okta" {
  source = "cyralinc/idp-okta/cyral"
  version = ">= 2.0.0"

  control_plane = "mytenant.cyral.com:8000"
  
  okta_app_name = "Cyral"
  
  idp_integration_name = "Okta"
}
```
With that being said, I think we should keep those variables to give a more flexible configuration in case a customer needs them. But in most cases, a simple configuration, as shown above, should be enough.